actiontext/app/helpers/action_text/tag_helper.rb

```diff
@@ -46,6 +46,22 @@ def render
       options = @options.stringify_keys
       add_default_name_and_id(options)
       options["input"] ||= dom_id(object, [options["id"], :trix_input].compact.join("_")) if object
+
+      translate_label = options.delete("label")
+
+      if !options.key?("aria-label") && translate_label
+        aria_label ||= ActionView::Helpers::Tags::Translator.new(
```

Comment:
It might be worth pulling some of this out into a helper method `add_default_aria_label(options)` for clarity (compare `add_default_name_and_id(options)`).